modules/ROOT/pages/how-to-guides/create-project.adoc

```diff
@@ -0,0 +1,126 @@
+= How to create a new project
```

Comment:
I believe this is more or less the same that we have in the Getting started, though? I have worked a bit on it in this https://github.com/neo4j/docs-graphql/pull/8PR